src/sentry/static/sentry/app/components/assistant/guideAnchor.tsx

```diff
@@ -116,13 +117,59 @@ const GuideAnchor = createReactClass<Props, State>({
     dismissGuide(currentGuide.guide, step, orgId);
   },
 
-  render() {
+  getHovercardExpBody() {
+    const {currentGuide, step} = this.state;
+
+    const totalStepCount = currentGuide.steps.length;
+    const currentStepCount = step + 1;
+    const currentStep = currentGuide.steps[step];
+    const lastStep = currentStepCount === totalStepCount;
+    const hasManySteps = totalStepCount > 1;
+
+    return (
+      <GuideExpContainer>
+        <GuideContent>
+          <GuideTitle>{currentStep.title}</GuideTitle>
+          <GuideDescription>{currentStep.description}</GuideDescription>
+        </GuideContent>
+        <GuideAction>
+          {lastStep ? (
+            <div>
+              <StyledButton size="small" onClick={this.handleFinish}>
+                {hasManySteps ? t('Enough Already') : t('Got It')}
+              </StyledButton>
+            </div>
+          ) : (
+            <div>
+              <DismissButton
+                priority="primary"
+                size="small"
+                onClick={this.handleDismiss}
+                href="#"
+              >
+                {t('Dismiss')}
+              </DismissButton>
+              <StyledButton size="small" onClick={this.handleNextStep}>
+                {t('Next')}
+              </StyledButton>
+            </div>
+          )}
+
+          {hasManySteps && (
+            <StepCount>{`${currentStepCount} OF ${totalStepCount}`}</StepCount>
```

Comment:
OF -- can we style with CSS instead?

Reply:
not sure i'm following. translated the `OF` though

Reply:
Ah I mean we should make it lower case and style the uppercase with css using text transform